src/utils/loadable.jsx

```diff
@@ -12,29 +11,14 @@ import Spinner from '../components/Spinner';
     marginTop: 60,
   },
 }))
-class Loading extends PureComponent {
-  content() {
-    const { error, timedOut, pastDelay } = this.props;
-
-    if (error) {
-      throw error;
-    } else if (timedOut || pastDelay) {
-      return <Spinner />;
-    }
-
-    return null;
-  }
-
+export default class Loading extends PureComponent {
   render() {
     const { classes } = this.props;
 
-    return <div className={classes.view}>{this.content()}</div>;
+    return (
+      <div className={classes.view}>
+        <Spinner />
+      </div>
+    );
   }
 }
-
-export default loader =>
-  Loadable({
-    loader,
-    loading: Loading,
-    timeout: 10000,
-  });
```

Comment:
We should be able to safely remove ` src/utils/loadable.jsx` assuming it is not used anymore.

Reply:
I think currently it is used in the Suspense part of `src/App/index.jsx`

Reply:
Oh, in that case we should use <Spinner /> directly instead of <Loading />. By the way, The Spinner component has a `loading` prop which would center the svg. That might remove the need of doing something like `textAlign: 'center'`.

Reply:
Agree! let me try 😄